ocd-1903.sql

```diff
@@ -0,0 +1,61 @@
+UPDATE openchpl.global_user_permission_map as gupm
+SET user_permission_id_user_permission = 2
+WHERE user_permission_id_user_permission = 3
+AND NOT EXISTS (SELECT user_id FROM openchpl.global_user_permission_map WHERE (user_id, 2) IN (SELECT user_id, user_permission_id_user_permission FROM openchpl.global_user_permission_map));
+
+UPDATE openchpl.global_user_permission_map as gupm
+SET user_permission_id_user_permission = 4
+WHERE user_permission_id_user_permission = 5
+AND NOT EXISTS (SELECT user_id FROM openchpl.global_user_permission_map WHERE (user_id, 4) IN (SELECT user_id, user_permission_id_user_permission FROM openchpl.global_user_permission_map));
+
+UPDATE openchpl.global_user_permission_map as gupm
+SET user_permission_id_user_permission = 2
+WHERE user_permission_id_user_permission = 3;
+
```

Comment:
It looks like lines 11-13 will do the updates that lines 1-4 did, but without the check that there aren't duplicates. Is that right?

Reply:
Yes for some reason 1-4 did not update users permissions when they only have 3 as their permission.